### ISSUE TITLE
Accepting Text for PGN imports

### ISSUE THREAD

**Issue by philipmiesbauer** [lichess-org/lichobile#2332]
**Describe the solution you'd like**
I regularly use lichess app to analyse games I played on chess.com. When I export PGN from chess.com then I need to first copy it into a different app and then copy it from there to paste it in lichess and I would like to be able to directly select lichess as app to export into and have the text in the PGN window directly.

Please let me know if there are more questions.

**Comment by jas14** [comment]
This looks like it might be a duplicate of #1403. Does that issue describe your request, @philipmiesbauer ?

**Comment by philipmiesbauer** [comment]
Not exactly. That request is already solved using the "Import game" function which is what I use, but I have to manually paste the PGN into it.

I hope for the import game activity to enable receiving the MIME type of data chess.com sends when "Exporting PGN".

![Screenshot_2023-03-23-06-39-57-365-edit_android.jpg](https://user-images.githubusercontent.com/13796773/227125224-d6b96ba7-0568-4c1c-9512-f96ae934cd9a.jpg)

The screenshot shows after I export a game of shows me all of the APAC that can accept the type of data, the share. I them open WhatsApp usually and select and copy the text from there.

Did that make sense?

Here is the Android documentation about the functionality https://developer.android.com/training/sharing/receive
